core/arch/arm/include/sm/sm.h

```diff
@@ -30,65 +31,83 @@
 
 #include <types_ext.h>
 
-struct sm_nsec_ctx {
```

Comment:
from `sm_save_modes_regs`, i think the registers are stored in opposite u32 order.
not an issue as all regs manipulated by 'sm' are outside `sm_mode_regs` structure.